Barotrauma/BarotraumaClient/ClientSource/Items/Components/Machines/Steering.cs

```diff
@@ -321,7 +321,7 @@ protected override void CreateGUI()
                         {
                             Vector2 vel = controlledSub == null ? Vector2.Zero : controlledSub.Velocity;
                             var realWorldVel = ConvertUnits.ToDisplayUnits(vel.Y * Physics.DisplayToRealWorldRatio) * 3.6f;
-                            return ((int)(-realWorldVel)).ToString();
+                            return MathF.Round(-realWorldVel,2).ToString("0.00");
```

Comment:
It's unnecessary to round the value first, "0.00" does that as well.

There should also be a space between the arguments passed to the Round method.

Reply:
That's really cool! I assumed it just truncated it. Brb.